### PR TITLE
Added lookaside.fbsbx.com domain

### PR DESCRIPTION
**Pull request by nemoload**
lookaside.fbsbx.com is used in displaying documents in Facebook groups

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const FACEBOOK_CONTAINER_NAME = "Facebook";
 const FACEBOOK_CONTAINER_COLOR = "blue";
 const FACEBOOK_CONTAINER_ICON = "briefcase";
 const FACEBOOK_DOMAINS = [
-  "facebook.com", "www.facebook.com", "fb.com", "fbcdn.net", "cdn.fbsbx.com",
+  "facebook.com", "www.facebook.com", "fb.com", "fbcdn.net", "cdn.fbsbx.com","lookaside.fbsbx.com",
```
Comment (NewAlexandria) [inline src/background.js]: The plugin uses a regex match, so it would be more-correct to have one `fbsbx.com` record, and remove any subdomain references

Comment (esaruoho) [reply src/background.js]: yeah, any duplication that's going on could, and should be simplified. that way we'll hopefully get @groovecoder to merge this into the repo.

Comment (esaruoho) [reply src/background.js]: @nemoload can you address @NewAlexandria 's comment in your PR?

Comment (nemoload) [reply src/background.js]: Sure, no problem.

**Comment by esaruoho** [comment]
@groovecoder @groovecoder-mozilla-owner looks like this could be landed?

**Comment by esaruoho** [comment]
apart from the missing space..

**Comment by groovecoder** [comment]
Since we don't have a dedicate QA team to check for bug regressions, can you provide an example of a request being made to `lookaside.fbsbx.com` from a 3rd-party site that can be spot-checked before and after this change?

Also would like to check if other *.fbsbx.com domains are used for other resources that could break significant website features.

**Comment by marczellm** [comment]
I think if a 3rd party site makes a request to fbsbx.com, it's not a bug in this extension if that breaks. It might be a bug in that website, but this extension's very job is to block such requests.

I find it highly unlikely though that such requests to Facebook's internal CDN (?) domain are very common. It's not a documented service they provide.

**Comment by groovecoder** [comment]
@marczellm do you have some specific URLs we can check for those assumptions?

**Comment by marczellm** [comment]
@groovecoder I'm arguing that most likely there are no such URLs, and even if there are, we shouldn't care.

**Comment by groovecoder** [comment]
Ah, I understand better now, and this seems like a good update. Very much like fbcdn.net - if someone receives a link straight to an asset on fbsbx.com, this will put them into Facebook Container when they visit it. Thanks for the patience and clarifications.

**Pull request merged**